src/main/java/org/broadinstitute/hellbender/utils/fermi/FermiLiteAssembler.java

```diff
@@ -15,35 +15,89 @@
  * Class that allows you to exercise Heng Li's fermi-lite assembler.
  * This is backed by JNI native code, which could fail to load.
  */
-public final class FermiLiteAssembler {
+public final class FermiLiteAssembler implements AutoCloseable {
     private static volatile boolean nativeLibLoaded = false;
+    private final ByteBuffer opts;
 
     public FermiLiteAssembler() {
         loadNativeLibrary();
+        opts = createDefaultOptions();
```

Comment:
Should there be an assert that the buffer you got is the expected size?  Is it likely that options will be added or removed and throw off the indexing?